microservices/object_detection/google_coral/annotactor.py

```diff
@@ -1,4 +1,5 @@
 from config import *
```

Comment:
What this class exactly does? Perhaps a one liner in the top would make things helpful.

Reply:
Having this type of imports is bad practice, and we should probably have explicit imports. For now we know more or less how things are but this doesn't scale. The issue with imports using `*` is that we are importing stuff we don't need and we could be bringing two functions with the same name without knowing, which can originate unwanted issues. There are a whole bunch of reasons to do explicit imports in python, but the "explicit is better than implicit" (https://www.python.org/dev/peps/pep-0020/) is very pythonic :)

Some general comments from the community here: https://stackoverflow.com/questions/9916878/importing-modules-in-python-best-practice

Also, from python guidelines, these wildcard imports are not recommended: https://www.python.org/dev/peps/pep-0008/#imports


I would change this already rather than later :)

Reply:
> What this class exactly does? Perhaps a one liner in the top would make things helpful.

It is an configuration file in python

Reply:
solved it